tests/e2e/vagrantdefaults.rb

```diff
@@ -34,6 +34,17 @@ def getInstallType(vm, release_version, branch)
   end
 end
 
+def addCoverageDir(vm, role)
+  service = role.include?("agent") ? "k3s-agent" : "k3s" 
+    script = <<~SHELL
+      mkdir -p /tmp/k3scov
+      mkdir -p /etc/systemd/system/#{service}.service.d
```

Comment:
You shouldn't need to create an override file, just put the env variable into one of the supported env files:
https://github.com/k3s-io/k3s/blob/2b24c9917cdfec92439ac68dd6706fffe20195a5/install.sh#L846-L848